### PR TITLE
Feature(sticky): add sticky pages

### PR DESCRIPTION
**Pull request by terry-ice**
增加了 sticky 页面

### CODE REVIEW

File: src/views/components-demo/sticky.vue [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    { key: 'b-platform', name: 'platformB' },
+    { key: 'c-platform', name: 'platformC' }
+  ]
+  private pickerOptions = {
+    disabledDate(time: Date) {
+      return time.getTime() > Date.now()
+    }
+  }
+}
+</script>
```
Comment (Armour) [inline src/views/components-demo/sticky.vue]:
```suggestion
</script>

```

File: src/views/components-demo/sticky.vue [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+<style lang="scss" scoped>
+.components-container div {
+  margin: 10px;
+}
+
+.time-container {
+  display: inline-block;
+}
+.el-button--mini {
```
Comment (Armour) [inline src/views/components-demo/sticky.vue]: is this css useful?

File: src/views/components-demo/sticky.vue [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+export default class StickyDemo extends Vue {
+  private time = Date
+  private url = ''
+  private platforms = ['a-platform']
+  private platformsOptions = [
+    { key: 'a-platform', name: 'platformA' },
+    { key: 'b-platform', name: 'platformB' },
+    { key: 'c-platform', name: 'platformC' }
+  ]
+  private pickerOptions = {
```
Comment (Armour) [inline src/views/components-demo/sticky.vue]: Add `:picker-options="pickerOptions"` to `<el-date-picker ... />`

File: src/views/components-demo/sticky.vue [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+      <div>placeholder</div>
+      <div>placeholder</div>
+      <div>placeholder</div>
+      <div>placeholder</div>
+      <div>placeholder</div>
+      <div>placeholder</div>
+      <div>placeholder</div>
+    </div>
+  </div>
+</template>
```
Comment (Armour) [inline src/views/components-demo/sticky.vue]:
```suggestion
</template>

```

File: src/components/Sticky/index.vue [hunk 5]
```diff
@@ -1,8 +1,10 @@
 <template>
```
Comment (Armour) [inline src/components/Sticky/index.vue]: Try to revert changes for this file, and run `yarn serve` to see the result, it should work already, you don't need to change this file.

File: src/views/components-demo/sticky.vue (hunk 2 above)
Comment (terry-ice) [reply src/views/components-demo/sticky.vue]: Control btn spacing

File: package.json [hunk 6]
```diff
@@ -52,6 +52,7 @@
   "devDependencies": {
     "@types/clipboard": "^2.0.1",
     "@types/codemirror": "^0.0.74",
+    "@types/dropzone": "^5.0.6",
```
Comment (Armour) [inline package.json]: Remove this.

File: src/views/components-demo/sticky.vue [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+</script>
+
+<style lang="scss" scoped>
+.components-container div {
+  margin: 10px;
+}
+
+.time-container {
+  display: inline-block;
+}
```
Comment (Armour) [inline src/views/components-demo/sticky.vue]:
```suggestion
}

```

**Comment by Armour** [comment]
#24

**Pull request merged**